README.md

```diff
@@ -6,116 +6,88 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/gardener/terminal-controller-manager)](https://goreportcard.com/report/github.com/gardener/terminal-controller-manager)
 [![reuse compliant](https://reuse.software/badge/reuse-compliant.svg)](https://reuse.software/)
 
-The `terminal-controller-manager` is used for the [webterminal feature](https://github.com/gardener/dashboard/blob/master/docs/Webterminals.md) of the [gardener/dashboard](https://github.com/gardener/dashboard) but is in general independant of the gardener and it's resource kinds like `Shoots`, `Seeds` etc.
-
+The `terminal-controller-manager` is used for the [webterminal feature](https://github.com/gardener/dashboard/blob/master/docs/Webterminals.md) of the [gardener/dashboard](https://github.com/gardener/dashboard).
 
 The `terminal-controller-manager` watches `Terminal` resources under the `dashboard.gardener.cloud/v1alpha1` API group and ensures the desired state on the host and target cluster.
 Host and target cluster can also be the same. For more details and a usage scenario [see docs here](https://github.com/gardener/dashboard/blob/master/docs/Webterminals.md).
 
 ## Development Setup
 
 Prerequisites:
-- [golang](https://golang.org/dl/) 
+- [golang](https://golang.org/dl/)
 - [kind](https://github.com/kubernetes-sigs/kind)
-- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+- [helm](https://helm.sh/docs/intro/install/)
```

Comment:
May be add that `go env GOPATH` must be in the `PATH`

Reply:
I have fixed the `hack/golangci-lint.sh` script to take the binary from the GOPATH https://github.com/gardener/terminal-controller-manager/pull/134/commits/57cf22f42608b01cf3e65fa04c6c84518645ae01
so either running this script or `make lint` should be sufficient